### DISCUSSION TITLE
Can I purchase and add vacuum to Create 3?

### DISCUSSION THREAD

**Discussion by fmorshed52** [iRobotEducation/create3_docs#349] [Questions] [answered]
I do see the "Can I turn my Create 3 robot into a vacuum?" in the Create 3 Q&A page, and the answer that the device does not come with the vacuum.  My question is:  Can I **purchase** and add the vacuum to my Create 3 device myself?

**Comment by shamlian** [comment, marked as answer]
The Create 3 educational robot is not designed to control any Roomba cleaning modules.
